palamedes/test_palamedes.py

```diff
@@ -0,0 +1,216 @@
+from palamedes import generate_variant_blocks, generate_alignment
```

Comment:
`generate_variant_blocks` is still defined in `align.py`, so the tests for it should stay in there. Only `generate_alignment` was moved.